docs/devElements.md

```diff
@@ -149,7 +149,7 @@ The different types of dependency properties currently available are summarized
 Property | Description
 --- | ---
 `coreStyles` | The styles required by this element, relative to `[PrairieLearn directory]/public/stylesheets`.
-`coreScripts` | The scripts required by this element, relative to `[PrairieLearn directory]/public/javascripts`.
+`coreScripts` | The scripts required by this element, relative to `[PrairieLearn directory]/public/javascripts`. Scripts in this directory are mainly used for compatibility with legacy elements and questions, and should be avoided if an alternative is available.
```

Comment:
I'd be happy to completely remove `coreScripts` from the docs, so that it only exists as an undocumented legacy feature. Or maybe we should be even stronger in our language and say `This directory is deprecated and should not be used.`

Reply:
I took the properties out of the table and added stronger warning. I also added the [`deprecated`](http://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.9.3) flag on the schemas.

Reply:
Nice!